src/index.js

```diff
@@ -21,10 +21,12 @@ import Request, {getNodeRequestOptions} from './request.js';
 import {FetchError} from './errors/fetch-error.js';
 import {AbortError} from './errors/abort-error.js';
 import {isRedirect} from './utils/is-redirect.js';
+import {FormData} from 'formdata-polyfill/esm.min.js';
 import {isDomainOrSubdomain} from './utils/is.js';
 import {parseReferrerPolicyFromHeader} from './utils/referrer.js';
 
-export {Headers, Request, Response, FetchError, AbortError, isRedirect};
+export {FormData, Headers, Request, Response, FetchError, AbortError, isRedirect};
+export * from 'fetch-blob/from.js';
```

Comment:
I'm a bit unsure abobut this wildcard export doe... what do u think?

Reply:
Hmm, yeah, I would _probably_ prefer to explicitly list them